INSTALLER.md

```diff
@@ -91,8 +98,157 @@ For code signing to work, you will need to set the `HF_PFX_FILE` and `HF_PFX_PAS
+    CPU Cores: 16
+    Minimum Disk Space: 40GB
+    ```
+2. Update the system and install dependencies.
+    ```
+    sudo yum update -y
+    sudo yum install git -y
+    sudo yum install rpm-build
+    ```
+3. Get and bootstrap Vircadia Builder.
+    ```
+    git clone https://github.com/vircadia/vircadia-builder.git
+    cd vircadia-builder
+    sudo ./install_amazon_linux_deps.sh
+    ```
+4. Run Vircadia Builder.
+    ```
+    ./vircadia-builder --build server
+    ```
+5. If Vircadia Builder needed to install dependencies and asks you to run it again then do so. Otherwise, skip to the next step.
+    ```
+    ./vircadia-builder --build server
+    ```
+6. Vircadia Builder will ask you to configure it to build the server. The values will be prefilled with defaults, the following steps will explain what they are and what you might want to put. *Advanced users: See [here](BUILD.md#possible-environment-variables) for possible environment variables and settings.*
+7. This value is the Git repository of Vircadia. You can set this URL to your fork of the Vircadia repository if you need to.
+    ```
+    Git repository: https://github.com/vircadia/vircadia/
+    # OR, for example
+    Git repository: https://github.com/digisomni/vircadia/
+    ```
+8. This value is the tag on the repository. If you would like to use a specific version of Vircadia, typically tags will be named like this: "v2021.1.0-rc".
+    ```
+    Git tag: master
+    # OR, for example
+    Git tag: v2021.1.0-rc
+    ```
+9. This value is the release type. For example, the options are `production`, `pr`, or `dev`. If you are making a build for yourself and others to use then use `production`.
+    ```
+    Release type: DEV
+    # OR, for example we recommend you use
+    Release type: PRODUCTION
+    ```
+10. This value is the release version. Release numbers typically should be in a format of `YEAR-MAJORVERSION-MINORVERSION` which might look like this: `2021.1.0`.
+    ```
+    Release number: 2021.1.0
+    ```
+11. This value is the build number. We typically use the hash of the most recent commit on that Git tag which might look like this: `fd6973b`.
+    ```
+    Build number: fd6973b
+    ```
+12. This value is the directory that Vircadia will get installed to. You should leave this as the default value unless you are an advanced user.
+    ```
+    Installation dir: /root/Vircadia
+    ```
+13. This value is the number of CPU cores that the Vircadia Builder will use to compile the Vircadia server. By default it will use all cores available on your build server. You should leave this as the default value it gives you for your build server.
+    ```
+    CPU cores to use for Vircadia: 16
+    ```
+14. This value is the number of CPU cores that the Vircadia Builder will use to compile Qt5 (a required component for Vircadia). By default it will use all cores available on your build server. You should leave this as the default value it gives you for your build server.
```

Comment:
```suggestion
14. This value is the number of CPU cores that the Vircadia Builder will use to compile Qt5 (a required component for Vircadia). By default it will use all cores available on your build server given you have enough memory. You should leave this as the default value it gives you for your build server.
```
It is pretty important that people don't just up the amount of threads to all available threads without having enough memory, as the build process might crash and/or the system might freeze.